ui/src/components/PickerDifficultyLevel.tsx

```diff
@@ -1,8 +1,8 @@
 import { MouseEventHandler } from 'react';
-import DifficultyLevel from './DifficultyLevel.tsx';
+import DifficultyLevel from './DifficultyLevel';
 import classNames from 'classnames';
-import { difficultyLevels } from '../consts/difficultyLevel.ts';
-import { useTranslation } from 'react-i18next';
+import { difficultyLevels } from 'consts/difficultyLevel';
+import { t } from 'i18next';
```

Comment:
To jak się korzysta z tego i18n: useTranslation czy import { t} ? Bo można to też tak zrobić...

Reply:
lepiej korzystać z t zwracanego przez hook useTranslation. Wtedy mamy pewność że po zmianie języka jest odpowiednia funkcja wywołwyana